pkg/api-server/testdata/resources/crud/put_meshaccesslog_00.golden.json

```diff
@@ -1 +1,5 @@
-{}
+{
+ "warnings": [
+  "MeshAccessLog type resource \"my-access-log\" has the following deprecated fields: "
+ ]
+}
```

Comment:
@gnana997 Hi, you can see there are no deprecation details in this sentence, as I said before. It's an obvious unexpected output, and we need to figure out why. I'll have a look at it later.